### PR TITLE
Added flag --layers to enable/disable functionality of showing each layers of image

### PR DESCRIPTION
**Pull request by kumaritanushree**

### CODE REVIEW

File: pkg/imgpkg/cmd/describe.go [hunk 1]
```diff
@@ -81,7 +84,7 @@ func (d *DescribeOptions) Run() error {
 	ttyEnabledLogger := util.NewUILevelLogger(logLevel, util.NewLoggerNoTTY(d.ui))
 	if d.OutputType == "text" {
 		p := bundleTextPrinter{logger: ttyEnabledLogger}
-		p.Print(description)
+		p.Print(description, d.Layers)
```
Comment (joaopapereira) [inline pkg/imgpkg/cmd/describe.go]: We are not providing this option for the yaml output?

File: pkg/imgpkg/cmd/describe.go [hunk 2]
```diff
@@ -138,14 +141,16 @@ func (p bundleTextPrinter) printerRec(description v1.Description, originalLogger
 		indentLogger.Logf("- Image: %s\n", b.Image)
 		indentLogger.Logf("  Type: Bundle\n")
 		indentLogger.Logf("  Origin: %s\n", b.Origin)
-		indentLogger.Logf("  Layers:\n")
-		for _, d := range b.Layers {
-			indentLogger.Logf("    - Digest: %s\n", d.Digest)
+		if layers {
```
Comment (joaopapereira) [inline pkg/imgpkg/cmd/describe.go]: instead of having a variable present can we jus change this if to check on `len(b.Layers)`?

**Review by joaopapereira — CHANGES_REQUESTED** [review]
Added some comments.

File: pkg/imgpkg/cmd/describe.go (hunk 1 above)
Comment (kumaritanushree) [reply pkg/imgpkg/cmd/describe.go]: In yaml with empty object it will not show the layers in output only.

File: pkg/imgpkg/cmd/describe.go (hunk 2 above)
Comment (kumaritanushree) [reply pkg/imgpkg/cmd/describe.go]: updated

**Review by joaopapereira — APPROVED** [review]
LGTM

**Pull request merged**